modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPInstanceLocalServiceImpl.java

```diff
@@ -1234,6 +1236,16 @@ public CPInstance updatePricingInfo(
 			BigDecimal cost, ServiceContext serviceContext)
 		throws PortalException {
 
+		BigDecimal maxValue = BigDecimal.valueOf(
+			CommercePriceConstants.COMMERCE_MAX_PRICE_VALUE);
+
+		if ((price.compareTo(maxValue) > 0) ||
```

Comment:
We are defining a min-price-value in the JSPs, but we are not validating it here.